### ISSUE TITLE
Cannot doc object destructuring

### ISSUE THREAD

**Issue by ozum** [tgreyuk/typedoc-plugin-markdown#56]
Hi,

I cannot create object destructuring object parameters in doc.

(I shortened for example purposes)
```ts
  /**
   * Some desc.
   * @param   value       Indexed value to search for.
   * @param   key         Param description
   * @param   fromIndex   The index etc.
   * @returns The first index of the element in the array; -1 if not found.
   */
  public getIndex(value: any, { key = this.defaultField, fromIndex = 0 }: { key?: keyof I; fromIndex?: number } = {}): number {
    return 1
  }
```

`key` and `fromIndex` is missing in docs.

### markdown output:
![image](https://user-images.githubusercontent.com/1497060/58113363-065a4680-7bfe-11e9-8298-7b683471b549.png)

### TypeDoc Theme output
![image](https://user-images.githubusercontent.com/1497060/58113394-12de9f00-7bfe-11e9-8bdf-689f3caf9501.png)

Kind Regards,

**Comment by tgreyuk** [comment]
Hi @ozum .. I am currently doing a bit of a refactor / re-write and will fix this with next release. Thanks.

**Comment by ozum** [comment]
@tgreyuk, being also developer of several open source libraries, thanks for the effort.

While you are re-writing, could you please `jsdoc2md`. I use it for JS projects and it has nice features. Having similar for TS would be great.

Thanks again,

**Comment by tgreyuk** [comment]
@ozum i have released a new version 2.0.0 and this should now be fixed.

Btw were there any particular features of jsdoc2md that you would like incorporated?

**Comment by ozum** [comment]
> Btw were there any particular features of jsdoc2md that you would like incorporated?

To not clutter this case, I opened a new one here for that: #59

**Comment by ozum** [comment]
> @ozum i have released a new version 2.0.0 and this should now be fixed.

Thanks a lot.
